tools/deploy_image_service.py

```diff
@@ -11,13 +13,17 @@
 SRC_FILE = os.path.join(os.getcwd(), 'deploy/assisted-image-service.yaml')
 DST_FILE = os.path.join(os.getcwd(), 'build', deploy_options.namespace, 'assisted-image-service.yaml')
 
+
 def main():
     utils.verify_build_directory(deploy_options.namespace)
 
     with open(SRC_FILE, "r") as src:
+        log.info(f"Loading source template file for assisted-image-service: {SRC_FILE}")
         raw_data = src.read()
         raw_data = raw_data.replace('REPLACE_NAMESPACE', f'"{deploy_options.namespace}"')
+        raw_data = raw_data.replace('REPLACE_IMAGE_SERVICE_IMAGE', os.environ.get("IMAGE_SERVICE"))
         data = yaml.safe_load(raw_data)
+        log.info(f"{data}")
```

Comment:
equivalent to this, no?
```suggestion
        log.info(data)
```